droidlet/lowlevel/robot_mover_utils.py

```diff
@@ -242,10 +242,72 @@ def visualize_examine(agent, robot_poses, object_xyz, label, obstacle_map, pts):
     robot_poses = np.asarray(robot_poses)
     plt.plot(robot_poses[:,0], robot_poses[:,1], 'r--')
 
-    pts = np.asarray(pts)
-    plt.plot(pts[:,0], pts[:,1], 'y--')
+    if gt_pts:
+        pts = np.asarray(gt_pts)
+        plt.plot(pts[:,0], pts[:,1], 'y--')
     
     # TODO: visualize robot heading 
     
     plt.savefig("{:04d}.jpg".format(vis_count))
-    vis_count += 1
\ No newline at end of file
+    vis_count += 1
+
+class ExaminedMap:
```

Comment:
I am ok with leaving this like this for the PR; but I am going to move to memory immediately after this is merged

Reply:
Sounds good. Added an issue here https://github.com/facebookresearch/fairo/issues/907